src/admin.js

```diff
@@ -0,0 +1,431 @@
+
+                  incomingWebhooks: () => ({
+                    list: () => api._get(`${appUrl}/services/${serviceId}/incomingWebhooks`),
+                    create: (data) => api._post(`${appUrl}/services/${serviceId}/incomingWebhooks`, data),
+                    incomingWebhook: (incomingWebhookId) => {
+                      const webhookUrl = `${appUrl}/services/${serviceId}/incomingWebhooks/${incomingWebhookId}`;
+                      return {
+                        get: () => api._get(webhookUrl),
+                        update: (data) => api._put(webhookUrl, data),
+                        remove: () => api._delete(webhookUrl)
+                      };
+                    }
+                  })
+
+                })
+              }),
+              pushNotifications: () => ({
+                list: (filter) => api._get(`${appUrl}/push/notifications`, filter),
+                create: (data) => api._post(`${appUrl}/push/notifications`, data),
+                pushNotification: (messageId) => ({
+                  get: () => api._get(`${appUrl}/push/notifications/${messageId}`),
+                  update: (data) => api._put(`${appUrl}/push/notifications/${messageId}`, data),
+                  setType: (type) => api._put(`${appUrl}/push/notifications/${messageId}/type`, { type }),
+                  remove: () => api._delete(`${appUrl}/push/notifications/${messageId}`)
+                })
+              }),
+              users: () => ({
+                list: (filter) => api._get(`${appUrl}/users`, filter),
+                create: (user) => api._post(`${appUrl}/users`, user),
+                user: (uid) => ({
+                  get: () => api._get(`${appUrl}/users/${uid}`),
+                  logout: () => api._put(`${appUrl}/users/${uid}/logout`),
+                  remove: () => api._delete(`${appUrl}/users/${uid}`)
+                })
+              }),
+              dev: () => ({
+                executePipeline: (body, userId, options) => {
+                  return api._post(
+                    `${appUrl}/dev/pipeline`,
+                    body,
+                    Object.assign({}, options, { user_id: userId }));
+                }
+              }),
+              authProviders: () => ({
+                list: () => api._get(`${appUrl}/auth_providers`),
+                create: (data) => api._post(`${appUrl}/auth_providers`, data),
+                authProvider: (providerId) => ({
+                  get: () => api._get(`${appUrl}/auth_providers/${providerId}`),
+                  update: (data) => api._patch(`${appUrl}/auth_providers/${providerId}`, data),
+                  enable: () => api._put(`${appUrl}/auth_providers/${providerId}/enable`),
+                  disable: () => api._put(`${appUrl}/auth_providers/${providerId}/disable`),
+                  remove: () => api._delete(`${appUrl}/auth_providers/${providerId}`)
+                })
+              }),
+              security: TODOnotImplemented,
```

Comment:
is there a ticket to implement this somewhere?

Reply:
I'm not sure, this file is copy pasted from its original location in `client.js`

Reply:
Just filed https://jira.mongodb.org/browse/STITCH-836 for this